static/js/settings/languages.js

```diff
@@ -0,0 +1,52 @@
+/*global each, category_template, $categories_form */
+window.addEventListener("DOMContentLoaded", function(){
+    language_template = document.querySelector("template#language_template").innerHTML;
+    $categories_form = $("form#languages tbody");
+});
+
+function add_language(event){
```

Comment:
Codacy found an issue: ['add_language' is defined but never used.](https://app.codacy.com/manual/saucebarbeque/Watcher3/pullRequest?prid=6225754)